src/hooks/usePrevious.test.ts

```diff
@@ -0,0 +1,32 @@
+import {renderHook} from '@testing-library/react-hooks';
+import {usePrevious} from '.';
+
+const setup = ({initialValue}: {initialValue?: any} = {}) =>
```

Comment:
Small `nit` but can we use `curly braces` and an explicit `return` whenever the return value falls to a new line? I believe this is somewhere in our Internal Code Standards